src/extension.ts

```diff
@@ -116,7 +124,10 @@ export async function activate(context: vscode.ExtensionContext): Promise<ApiImp
 	}
 
 	context.subscriptions.push(registerBuiltinGitProvider(apiImpl));
-	context.subscriptions.push(registerLiveShareGitProvider(apiImpl));
+	const liveshareGitProvider = registerLiveShareGitProvider(apiImpl);
+	context.subscriptions.push(liveshareGitProvider);
+	const liveshareApi = liveshareGitProvider.initialize();
```

Comment:
`await` here?

Reply:
the await (actually a .then()) is done later on th einit method. I will rename the name to 'liveshareApiPromise'